app/controllers/admissions_admin/interviews_controller.rb

```diff
@@ -76,8 +76,11 @@ def show_warning_if_other_interviews_take_place_within_30_minutes
     @interview.job_application.applicant.job_applications.each do |application|
       next if (application == @interview.job_application) || application.interview.nil? || application.interview.time.nil?
 
-      time_interval = ((application.interview.time - 29.minutes)..(application.interview.time + 29.minutes))
-      next unless time_interval.include? @interview.time
+      start_date = application.interview.time - 29.minutes
+      interview_time = application.interview.time
```

Comment:
This should be @interview.time, or else the next unless statement will always be true.
@interview is the interview that is being updated, application.interview is the current interview in the list of interviews being iterated over

Reply:
Whops, my bad – thanks!